scripts/import_cloudwatch_log_groups.py

```diff
@@ -0,0 +1,83 @@
+import json
+
+import boto3
+from more_itertools import (
+    one,
+)
+
+from azul import (
+    config,
+    logging,
+)
+from azul.logging import (
+    configure_script_logging,
+)
+from azul.terraform import (
+    terraform,
+)
+
+log = logging.getLogger(__name__)
+
+
+def main():
+    def add_prefix(name):
+        return 'aws_cloudwatch_log_group.' + name
+
+    log_groups = {}
+    tf_component = config.terraform_component
+
+    if tf_component == 'shared':
+        if config.slack_integration:
+            # Chatbot log groups
+            name = add_prefix('chatbot')
+            log_group = '/aws/chatbot/' + config.qualified_resource_name('chatbot')
+            log_groups[name] = log_group
+    elif not tf_component:
+        # Lambda function log groups
+        tf_path = f'{config.project_root}/terraform/api_gateway.tf.json'
```

Comment:
```suggestion
        tf_path = config.project_root + '/terraform/api_gateway.tf.json'
```

https://github.com/DataBiosphere/azul/blob/develop/CONTRIBUTING.rst#string-interpolation